src/config.c

```diff
@@ -1045,6 +1082,16 @@ struct rewriteConfigState *rewriteConfigReadOldFile(char *path) {
             continue;
         }
 
+        /* If the config file contains a configuration that is no longer present
+         * in the configs array: remove it from the .conf file. Use case right now
+         * is module with configurations unloaded after a rewrite with the module configs. */
+        if (!lookupConfig(argv[0])) {
```

Comment:
@nichchun There is CI fail in the fully CI of this PR: https://github.com/sundb/redis/runs/5655135034?check_suite_focus=true

```sentinel down-after-milliseconds setmaster 20000```
At this time, only using `argv[0]` cannot determine whether the configuration already exists.

```
Testing unit: 00-base.tcl
05:09:46> (start-init) Flush config and compare rewrite config file lines: FAILED: Expected 0 (context: type eval line 9 cmd {assert [string equal $line1 $line2]} proc ::foreach_instance_id)
(Jumping to next unit after error)
FAILED: caught an error in the test 
assertion:Expected 0 (context: type eval line 9 cmd {assert [string equal $line1 $line2]} proc ::foreach_instance_id)
    while executing
"foreach_sentinel_id id {
        assert_match "OK" [S $id SENTINEL FLUSHCONFIG]
        set file1 ../tests/includes/sentinel.conf
        set file2 [f..."
    ("uplevel" body line 2)
    invoked from within
"uplevel 1 $code"
Testing unit: 01-conf-update.tcl
```